tbx/static_src/sass/components/_youtube_embed.scss

```diff
@@ -0,0 +1,66 @@
+@use 'config' as *;
+
+.youtube-video-container {
```

Comment:
We'll want to adapt these to use BEM but i can do that as part of the front-end work.

Reply:
sorting in front-end ticket.